lib/ansible/plugins/filter/to_nice_json.yml

```diff
@@ -47,9 +47,17 @@ DOCUMENTATION:
       description: Specifies an indentation level. Passed to an underlying C(json.dumps) call.
       default: 4
       type: int
+    separators:
+      description: |
+        A tuple of (item_separator, key_separator) strings to use when serializing.
+        If not specified, defaults to (',', ': ') for nice formatting.
+        Common alternatives are (',', ':') for compact output or custom separators for special formatting needs.
+      default: "(',', ': ')"
```

Comment:
The default is not a string, so this should not be quoted. Yes, the existing docs for `to_json` are incorrect.